pkgs/applications/networking/mailreaders/afew/default.nix

```diff
@@ -15,19 +15,22 @@ pythonPackages.buildPythonApplication rec {
     pythonPackages.notmuch chardet dkimpy
   ] ++ stdenv.lib.optional (!pythonPackages.isPy3k) subprocess32;
 
+  makeWrapperArgs = [
+    ''--prefix PATH ':' "${notmuch}/bin"''
+  ];
+
+  outputs = [ "out" "doc" ];
+
   postBuild =  ''
-    make -C docs man
+    python setup.py build_sphinx -b html,man
   '';
 
   postInstall = ''
-    mandir="$out/share/man/man1"
-    mkdir -p "$mandir"
-    cp docs/build/man/* "$mandir"
```

Comment:
does that mean there is no more terminal documentation for afew and just an html document?